src/aws_credentials.erl

```diff
@@ -163,7 +187,7 @@ setup_update_callback(Expires) when is_integer(Expires) ->
 
 -spec setup_callback(pos_integer()) -> reference().
 setup_callback(Seconds) ->
-    erlang:send_after(Seconds*1000, self(), refresh_client).
+    erlang:send_after(Seconds*1000, self(), refresh_credentials).
```

Comment:
I guess that's what it was meant to be, right? Otherwise the message would not be handled in the `handle_info` at all.

Reply:
Yeah I don't know why I never noticed this before! Derp. Thanks for fixing it.